docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Expose multiple services on the same host
+---
+
+This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
+
+You can either define a service at the root level, which is applied to all paths except the ones you've explicitly set service for at the rules level, or you can just define different services on each path separately without the need to define a root service.
+
+The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create those workloads, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
+
+
+## Multiple services definition on different paths
+
+Follow the instruction to expose your unsecured instance of the HttpBin service and your unsecured sample Function at the same time on different paths.
+
+1. Export the following value as an environment variable:
+
+   ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} # This is a Kyma domain or your custom subdomain e.g. api.mydomain.com.
+   ```
+
+2. To expose the instance of the HttpBin service and the instance of the sample Function, create an API Rule CR in your Namespace. If you don't want to use Kyma's default gateway, replace `kyma-system/kyma-gateway` with your custom gateway. Run:
```

Comment:
i would suggest adding a note here that purpose of this tutorial is to highlight multiple services per path and recommend securing workload and read https://kyma-project.io/docs/kyma/latest/03-tutorials/00-api-exposure/apix-03-expose-and-secure-workload-oauth2/ or https://kyma-project.io/docs/kyma/latest/03-tutorials/00-api-exposure/apix-05-expose-and-secure-workload-jwt/. the same note should be added for 2nd part of this tutorial